TestFoundation/FixtureValues.swift

```diff
@@ -126,6 +126,28 @@ enum Fixtures {
         return dif
     }
     
+    // ===== ISO8601DateFormatter =====
+    
+    static let iso8601FormatterDefault = TypedFixture<ISO8601DateFormatter>("ISO8601DateFormatter-Default") {
+        
+        let idf = ISO8601DateFormatter()
+        idf.timeZone = Calendar.neutral.timeZone
+        
+        return idf
+        
```

Comment:
Other functions don't have blank lines after the opening and before the closing braces.